test/src/main/resources/scala/test/more-hellos

```diff
@@ -0,0 +1,2 @@
+More Hello
```

Comment:
this file is not in the `exports_files` is that intentional? Can you add a comment in the `BUILD` as to why?